common/lib/xmodule/xmodule/tests/test_course_metadata_utils.py

```diff
@@ -159,6 +160,10 @@ def mock_strftime_localized(date_time, format_string):
                     advertised_start_unparsable.title()
                 ),
                 TestScenario(
+                    (test_datetime, advertised_start_bad_date, 'DATE_TIME', ugettext, mock_strftime_localized),
```

Comment:
Update for future readers: I did not add more tests, but instead added comments that explained why the existing tests were sufficient.